core/templates/domain/exploration/SubtitledUnicodeObjectFactory.ts

```diff
@@ -0,0 +1,87 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Factory for creating new frontend instances of SubtitledUnicode
+ * domain objects.
+ */
+
+export interface ISubtitledUnicodeBackendDict {
+  'content_id': string;
+  'unicode_str': string;
+}
+
+import { Injectable } from '@angular/core';
+import { downgradeInjectable } from '@angular/upgrade/static';
+
+export class SubtitledUnicode {
+  _unicode: string;
+  _contentId: string;
```

Comment:
```suggestion
  private _unicode: string;
  private _contentId: string;
```

But I do not understand why do these properties need to be private?

/cc @seanlip @vojtechjelinek @ankita240796

Reply:
Reverted - Done

Reply:
I'm a bit confused. It doesn't seem to be reverted?

I'm fine with them being private though, in general we should make things private unless they should actually be exposed.

Reply:
Mb, seems I was a little confused with the convention- didn't fully realize prefixing with _ indicates that it is private (even tho it can be publicly accessed). Keeping as is for now.

Reply:
OK, fine either way, though it is OK to add "private" before it IMO. It seems like a better practice.